server/src/test/java/org/elasticsearch/index/mapper/DynamicMappingTests.java

```diff
@@ -436,106 +431,6 @@ public void testComplexArray() throws Exception {
                 .endObject().endObject().endObject()), serialize(update));
     }
 
-    public void testReuseExistingMappings() throws IOException, Exception {
```

Comment:
This seems like a pretty useful test, and could make sense even when there is only one type. Instead of deleting it, we could validate that the `update` variable below is `null`?

Reply:
++ I re-added the test and updated it accordingly.